test/integration/esys-create-session-auth.int.c

```diff
@@ -80,6 +80,12 @@ test_esys_create_session_auth(ESYS_CONTEXT * esys_context)
         .buffer = {1, 2, 3, 4, 5}
     };
 
+#ifdef TEST_LARGE_AUTH
+    for (int i = 0; i < 33; i++)
```

Comment:
shouldn't we really push these over size of TPMU_HA?

Reply:
I would prefer to use 33 in this case, because it's the first size greater than the size of the name alg.

Reply:
Yep you should try 1 over the HashAlg size and 1 over TPMU_HA size to trip that other condition for Esys_TR_SetAuth.

Reply:
Disregard again.